routes/v1/external.js

```diff
@@ -12,8 +12,11 @@ const axios = require('axios');
 const agent = new SocksProxyAgent(`socks5h://${constants.TOR_PROXY_IP}:${constants.TOR_PROXY_PORT}`);
 
 router.get('/price', auth.jwt, safeHandler(async(req, res) => {
+  // Default to USD
+  const sym = req.params.sym || "USD";
```

Comment:
`sym` is nice and short, although we can probably rename it to something that's more verbal, like `currency`:

```suggestion
  const currency = req.params.currency || "USD";
```